### PR TITLE
Xsup 39289 auto resolve

### PR DESCRIPTION
**Pull request by maimorag**
## Related Issues
fixes: [XSUP-39289](https://jira-dc.paloaltonetworks.com/browse/XSUP-39289)

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

**Review by YuvHayun — APPROVED** [review]
Nice catch!

File: Packs/Core/ReleaseNotes/3_0_59.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Indicators detection
+
+- Updated the CoreIRApiModule with support for for status ***resolved_auto_resolve***  close-reason in XSOAR-XDR close-reasons mapping.
```
Comment (eyalpalo) [inline Packs/Core/ReleaseNotes/3_0_59.md:6]:
```suggestion
- Updated the CoreIRApiModule to support status ***resolved_auto_resolve***  close-reason in XSOAR-XDR close-reasons mapping.
```

File: Packs/Core/ReleaseNotes/3_0_59.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+##### Investigation & Response
+
+- Updated the CoreIRApiModule with support for for status ***resolved_auto_resolve***  close-reason in XSOAR-XDR close-reasons mapping.
```
Comment (eyalpalo) [inline Packs/Core/ReleaseNotes/3_0_59.md:10]: Same as above

File: Packs/CortexXDR/ReleaseNotes/6_1_68.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Added support for status ***resolved_auto_resolve***  close-reason in XSOAR-XDR close-reasons mapping.
```
Comment (eyalpalo) [inline Packs/CortexXDR/ReleaseNotes/6_1_68.md:6]: Same as above

File: Packs/ctf01/ReleaseNotes/1_0_27.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Cortex XDR - IR CTF
+
+- Updated the CoreIRApiModule with support for for status ***resolved_auto_resolve***  close-reason in XSOAR-XDR close-reasons mapping.
```
Comment (eyalpalo) [inline Packs/ctf01/ReleaseNotes/1_0_27.md:6]: Same as above

**Review by eyalpalo — CHANGES_REQUESTED** [review]

**Review by eyalpalo — APPROVED** [review]

**Pull request merged**